ocfweb/docs/docs/staff/policies/lab-reservation-policy.md

```diff
@@ -0,0 +1,16 @@
+[[!meta title="Lab Reservation Policy"]]
+
```

Comment:
Should also have an annotation describing who established the policy (in this case, the GM) and when.